docs/api-ref.md

```diff
@@ -3742,7 +3742,7 @@ Deletes a workbook with the specified ID.
 To specify the site, create a `TableauAuth` instance using the content URL for the site (`site_id`), and sign in to that site.  See the [TableauAuth class](#tableauauth-class).  
 
 
-REST API: [Delete Workbook](https://onlinehelp.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref.htm#Delete_Workbook31){:target="_blank"}  
+REST API: [Delete Workbook](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref.htm#Delete_Workbook31)
```

Comment:
This url should be: https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref.htm#delete_workbook1

Reply:
done, good catch